prow/config.yaml

```diff
@@ -2147,9 +2244,10 @@ presubmits:
   - name: pull-kubernetes-e2e-kops-aws
     agent: kubernetes
     skip_branches:
-    - release-1.6  # runs on Jenkins?
+    - release-1.6  # runs on Jenkins? NOT ANYMORE :D
```

Comment:
we can probably just get rid of 1.6 now entirely, right?

Reply:
probably? that will be a pretty big, separate PR. we have this in a ton of places